src/info.rs

```diff
@@ -262,6 +273,25 @@ impl Info {
         authors
     }
 
+    fn get_git_info() -> (String, String){
+        let version = Command::new("git")
+            .arg("--version")
+            .output()
+            .expect("Failed to execute git.");
+        let version = String::from_utf8_lossy(&version.stdout);
+        let version = version.to_string().replace('\n',"");
+
+        let username = Command::new("git")
+            .arg("config")
+            .arg("--get")
+            .arg("user.name")
```

Comment:
Good job!
But it looks like this will get `user.name` in the *current directory*, not the directory that could get passed by `onefetch -d path/to/project`. This could be an issue when the local `user.name` is different from the global `user.name`.
See `get_authors` for an example of running a git command for the specified directory.

Reply:
Done, i've tested with the -d flag and seems fine now, can you verify?

Reply:
Looks good to me :+1: